source/whats-new.txt

```diff
@@ -4,4 +4,136 @@
+* :ref:`Version 5.0 <version-5.0>`
+
+.. _kotlin-sync-version-5.1.3:
+
+What's New in 5.1.3
+-------------------
+
+The 5.1.3 driver patch release includes the following changes:
+
+.. sharedinclude:: dbx/jvm/v5.1.3-wn-items.rst
+
+.. _kotlin-sync-version-5.1.2:
+
+What's New in 5.1.2
+-------------------
+
+The 5.1.2 driver patch release includes the following changes:
+
+- Support for encoding Kotlin data classes with nullable
+  generic parameter types. For example, you can encode the ``Container`` class
+  in the following code:
+  
+  .. code-block:: kotlin
+
+     @Serializable
+     data class Box<T>(
+         val boxed: T
+     )
+     
+     @Serializable
+     data class Container(
+         val box: Box<String?>
+     )
+
+.. _kotlin-sync-version-5.1.1:
+
+What's New in 5.1.1
+-------------------
+
+The 5.1.1 driver patch release includes the following changes:
+
+- When using the ``MONGODB-OIDC`` authentication mechanism, you must not
+  include comma characters in the ``authMechanismProperties`` connection
+  string value. To learn more about this behavior, see the
+  :ref:`MONGODB-OIDC <kotlin-oidc>` section of the Enterprise
+  Authentication guide.
+
+.. _kotlin-sync-version-5.1:
+
+What's New in 5.1
+-----------------
+
+.. warning:: Deprecations in this release
+
+   To avoid breaking changes in future major releases of the driver,
+   replace any application code that depends on deprecated program elements.
+
+This section includes the following information:
+
+- :ref:`kotlin-deprecations-5.1`
+- :ref:`kotlin-improvements-5.1`
+- :ref:`kotlin-new-features-5.1`
+
+.. _kotlin-deprecations-5.1:
+
+Deprecations in 5.1
+~~~~~~~~~~~~~~~~~~~
+
+- Support for {+mdb-server+} v3.6 is deprecated and will be removed in the
+  next driver version release. To learn how to upgrade your {+mdb-server+}
+  deployment, see :manual:`Release Notes </release-notes>` in the {+mdb-server+}
+  manual.
+
+.. _kotlin-improvements-5.1:
+
+Improvements in 5.1
+~~~~~~~~~~~~~~~~~~~
+
+- Internal testing of GraalVM native image technology. These tests involve building
+  native applications by using the GraalVM native-image tool.
+
+- Enhanced support for the ``MONGODB-OIDC`` authentication mechanism.
+  To learn more about OIDC, see the :ref:`kotlin-oidc` section of the
+  Enterprise Authentication Mechanisms guide.
+
+- Fixes an issue in which operations used the incorrect codec when using
+  a polymorphic ``MongoCollection`` instance. This ensures that
+  discriminator information is not lost when using ``bson-kotlinx``.
+
+- Fixes an issue in which the class discriminator was the first field
+  when decoding, resulting in field type errors when using a polymorphic
+  ``MongoCollection`` instance.
+
+.. _kotlin-new-features-5.1:
+
+New Features in 5.1
+~~~~~~~~~~~~~~~~~~~
+
+- Support for polymorphic serialization. To learn more, see the
+  :ref:`kotlin-polymorphic` section of the Kotlin Serialization guide.
```

Comment:
```suggestion
- Support for polymorphic serialization. 
```
Don't see a ticket for this page. Are we planning on making a serialization page, or should I delete this sentence?

Reply:
I'll make a ticket for a serialization page, i do think it is important

Reply:
https://jira.mongodb.org/browse/DOCSP-42666

Reply:
Can you comment this out as a TODO then?